src/main/java/com/kashdeya/tinyprogressions/items/materials/ArmorMaterialTier.java

```diff
@@ -21,10 +21,10 @@
 	FLINT   ("flint"   , 4 ,  new int[]{1, 3, 3, 1}  , 10,  SoundEvents.ITEM_ARMOR_EQUIP_GENERIC, 0.01F, () -> { return Ingredient.fromItems(Items.FLINT);  }, 0F),
 	BONE    ("bone"    , 3  ,  new int[]{1, 3, 2, 1}  , 8 ,  SoundEvents.ITEM_ARMOR_EQUIP_GENERIC, 0.02F, () -> { return Ingredient.fromItems(Items.BONE); }, 0F),
 	WOOD    ("wood"    , 2  ,  new int[]{1, 2, 1, 1}  , 5 ,  SoundEvents.ITEM_ARMOR_EQUIP_GENERIC, 0.01F, () -> { return Ingredient.fromItems(Items.ACACIA_WOOD, Items.BIRCH_WOOD, Items.DARK_OAK_WOOD, Items.JUNGLE_WOOD, Items.OAK_WOOD, Items.SPRUCE_WOOD); }, 0F),
-	LAVA    ("lava"    , 50 ,  new int[]{5, 8, 10, 5} , 35,   SoundEvents.ITEM_ARMOR_EQUIP_DIAMOND, 5.0F,  () -> { return Ingredient.fromItems(TechBlocks.lava_block.get(), TechBlocks.nether_lava_block.get(),TechBlocks.lava_infused_stone.get()); }, 1F),
-	WITHER  ("wither"  , 100,  new int[]{6, 9, 11, 6} , 40,  SoundEvents.ITEM_ARMOR_EQUIP_DIAMOND, 15.0F, () -> { return Ingredient.fromItems(TechItems.wither_rib.get()); }, 6F),
-	DRAGON  ("dragon"  , 200,  new int[]{7, 10, 12, 7}, 75,  SoundEvents.ITEM_ARMOR_EQUIP_DIAMOND, 20.0F, () -> { return Ingredient.fromItems(TechItems.dragon_scale.get()); }, 8F),
-	OBSIDIAN("obsidian", 33 ,  new int[]{3, 6, 8, 3}  , 10,   SoundEvents.ITEM_ARMOR_EQUIP_DIAMOND, 3.0F,  () -> { return Ingredient.fromItems(Items.OBSIDIAN); }, 2F),
+	LAVA    ("lava"    , 50 ,  new int[]{5, 8, 10, 5} , 35,   SoundEvents.ITEM_ARMOR_EQUIP_DIAMOND, 5.0F,  () -> { return Ingredient.fromItems(TechBlocks.lava_block.get(), TechBlocks.nether_lava_block.get(),TechBlocks.lava_infused_stone.get()); }, 0.1F),
+	WITHER  ("wither"  , 100,  new int[]{6, 9, 11, 6} , 40,  SoundEvents.ITEM_ARMOR_EQUIP_DIAMOND, 15.0F, () -> { return Ingredient.fromItems(TechItems.wither_rib.get()); }, 0.6F),
+	DRAGON  ("dragon"  , 200,  new int[]{7, 10, 12, 7}, 75,  SoundEvents.ITEM_ARMOR_EQUIP_DIAMOND, 20.0F, () -> { return Ingredient.fromItems(TechItems.dragon_scale.get()); }, 2F),
+	OBSIDIAN("obsidian", 33 ,  new int[]{3, 6, 8, 3}  , 10,   SoundEvents.ITEM_ARMOR_EQUIP_DIAMOND, 3.0F,  () -> { return Ingredient.fromItems(Items.OBSIDIAN); }, 0.2F),
```

Comment:
You said toughness balance, but I think you meant knockback resistance.  Just wanted to make sure I was on the same page. Cause the toughness values were not changed, but knockback was added and changed. Knockback works on a percentage bases.. so 0.01F is 1% where 1.0F is 100%, if I am not mistaken. I believe netherite armor is 10% each piece of armor for reference.